python/test/unit/fem/test_assembler.py

```diff
@@ -499,3 +499,17 @@ def monitor(ksp, its, rnorm):
     ksp.solve(b2, x2)
     assert ksp.getConvergedReason() > 0
     assert x0.norm() == pytest.approx(x2.norm(), 1e-8)
+
+
+def test_projection():
+    mesh = dolfin.UnitCubeMesh(dolfin.MPI.comm_world, 4, 4, 4)
+    V = dolfin.function.FunctionSpace(mesh, ("CG", 1))
+
+    x = ufl.SpatialCoordinate(mesh)
+    expr = x[0] ** 2
+
+    f = dolfin.project(expr, V)
+    integral = dolfin.fem.assemble(f * ufl.dx)
+
+    integral_analytic = 1.0 / 3
+    assert numpy.isclose(integral_analytic, integral, rtol=1.e-6, atol=1.e-12)
```

Comment:
Is there a reason why `numpy.isclose` is required, rather than  `pytest.approx`?